website/src/pages/en/subgraphs/querying/_meta.js

```diff
@@ -1,7 +1,7 @@
 import titles from './_meta-titles.json'
 
 export default {
-  introduction: '',
+  introduction: 'How to Query',
```

Comment:
```suggestion
  introduction: '',
```
Page titles should never be set in `_meta.js` files, but directly in the MDX's frontmatter, either as `title` or as `sidebarTitle` (as you've also done!), so that they remain translatable.